src/client/pages/SignIn.stories.tsx

```diff
@@ -114,3 +106,10 @@ export const WithJobsAndSocialSigninBlocked = (args: SignInProps) => (
 WithJobsAndSocialSigninBlocked.story = {
 	name: 'with Jobs terms and social sign-in blocked',
 };
+
+export const isReauthenticate = (args: SignInProps) => (
```

Comment:
Stories should use PascalCase.

Just saw this:
![Screenshot 2024-06-18 at 09 42 51](https://github.com/guardian/gateway/assets/13315440/90ce0eee-3cb2-45c7-a171-469928b5d17a)

Reply:
Weird that this slipped through!